themes/vue/layout/search-page.ejs

```diff
@@ -0,0 +1,160 @@
+<div id="search-page">
+    <form class="search-form">
+        <input
+            class="search-query"
+            v-model="search"
+            placeholder="Search VueJS"
+        >
+        <p v-if="totalResults">
+            <strong>{{ totalResults }} results</strong> found in {{ queryTime }}ms
+        </p>
+    </form>
+
+    <template v-if="results.length">
+        <search-result
+            v-for="(result, i) in results"
+            :key="i"
+            :result="result"
+        ></search-result>
+    </template>
+
+    <p v-else>No results were found.</p>
+
+    <div ref="infiniteScrollAnchor"></div>
+
+</div>
+
+<script src="//cdn.jsdelivr.net/algoliasearch/3/algoliasearch.min.js"></script>
+<script src="//cdn.jsdelivr.net/algoliasearch.helper/2/algoliasearch.helper.min.js"></script>
+<script src="//polyfill.io/v2/polyfill.min.js?features=IntersectionObserver"></script>
+<script>
+var match = window.location.pathname.match(/^\/(v\d+)/)
+var version = match ? match[1] : 'v2'
+var algolia = algoliasearch('BH4D9OD16A', '85cc3221c9f23bfbaa4e3913dd7625ea')
+var algoliaHelper = algoliasearchHelper(algolia, 'vuejs', {
+    hitsPerPage: 15,
```

Comment:
We use 2-space indentation (both JS and HTML) for our docs, can you please check?